pkg/injector/bootstrap.go

```diff
@@ -30,16 +31,16 @@ func (wh *mutatingWebhook) createEnvoyBootstrapFromExisting(newBootstrapSecretNa
 		return nil, fmt.Errorf("error unmarshalling envoy bootstrap config: %w", err)
 	}
 
-	config.Node.Id = cert.GetCommonName().String()
+	config.Node.Id = proxyUUID.String()
 
-	return wh.marshalAndSaveBootstrap(newBootstrapSecretName, namespace, config, cert)
+	return wh.marshalAndSaveBootstrap(bootstrapConfigName(proxyUUID), namespace, config, cert)
 }
 
-func (wh *mutatingWebhook) createEnvoyBootstrapConfig(name, namespace, osmNamespace string, cert *certificate.Certificate, originalHealthProbes models.HealthProbes) (*corev1.Secret, error) {
+func (wh *mutatingWebhook) createEnvoyBootstrapConfig(proxyUUID uuid.UUID, namespace string, cert *certificate.Certificate, originalHealthProbes models.HealthProbes) (*corev1.Secret, error) {
 	builder := bootstrap.Builder{
-		NodeID: cert.GetCommonName().String(),
+		NodeID: proxyUUID.String(),
```

Comment:
I don't know what the implications of this change would be but should this be a separate PR in case there it is need to change?  would make this current change a smaller change set too.

Reply:
I have https://github.com/openservicemesh/osm/pull/5086, happy to submit that first